Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml

```
@@ -291,11 +291,11 @@
     - MagazineACP14
   - type: BallisticAmmoProvider
     mayTransfer: true
-    proto: CartridgePistol  # 57x28
+    proto: CartridgeMagnum  # 57x28
     whitelist:
       tags:
-      - CartridgePistol  # 57x28
-    capacity: 15
+      - CartridgeMagnum  # 57x28
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)